content/en/docs/ops/deployment/performance-and-scalability/index.md

```diff
@@ -83,22 +83,21 @@ The latency, throughput, and the proxies' CPU and memory consumption are measure
 ### CPU and memory
 
 Since the sidecar proxy performs additional work on the data path, it consumes CPU
-and memory. As of Istio 1.7, a proxy consumes about 0.5 vCPU per 1000
+and memory. In Istio {{< istio_release_name >}}, a proxy consumes about 0.5 vCPU per 1000
 requests per second.
 
 The memory consumption of the proxy depends on the total configuration state the proxy holds.
 A large number of listeners, clusters, and routes can increase memory usage.
-Istio 1.1 introduced namespace isolation to limit the scope of the configuration sent
-to a proxy. In a large namespace, the proxy consumes approximately 50 MB of memory.
+In a large namespace with [namespace isolation](/docs/reference/config/networking/sidecar/) enabled, the proxy consumes approximately 50 MB of memory.
 
 Since the proxy normally doesn't buffer the data passing through,
 request rate doesn't affect the memory consumption.
 
 ### Latency
 
 Since Istio injects a sidecar proxy on the data path, latency is an important
-consideration. Istio adds an authentication filter and a telemetry filter and a metadata exchange filter
-to the proxy. Every additional filter adds to the path length inside the proxy and affects latency.
```

Comment:
> Every feature Istio adds

This is meant to say "features that add filters to Envoy".  I'm worried people might read it to say "if I just configure an existing feature, or I configure something that is built into Envoy, it will make Istio slower".